generators/server/templates/src/main/java/package/config/_ExcludedFromComponentScan.java

```diff
@@ -0,0 +1,9 @@
+package <%=packageName%>.config;
+
+/**
+ * a simple annotation, to be used for easy exclusion from component scan
+ *
+ * @author David Steiman
+ */
```

Comment:
The tools from spring boot actually allow to define, which annotations lead to bean exclusion. Using classes in exclude filter is good, if it's the only component to be excluded. As soon there are more of them, it just looks ugly in my opinion.

I just thought using some annotation like this would be a more elegant way. The only downside may be, you don't see excluded beans by looking on app class (but a quick search helps here...)